### PR TITLE
Add side navigation to documentation, update URLs

### PR DESCRIPTION
**Pull request by zachmargolis**
This makes OpenID connect a peer to SAML, and makes it publicly navigable

---

**Why**: Clearer organization and structure

* Also adds redirect URLs

screen shot 2017-02-24 at 11 30 57 am

### CODE REVIEW

File: _includes/sidenav.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <a class="{% if page.url == '/attributes/' %}usa-current{% endif %}" href="{{ site.baseurl }}/attributes">User attributes</a>
+  </li>
+  <li>
+    <a class="{% if page.url == '/register/' %}usa-current{% endif %}" href="{{ site.baseurl }}/register">Register your service provider</a>
+  </li>
+  <li>
+    <a class="{% if page.url == '/saml/' %}usa-current{% endif %}" href="{{ site.baseurl }}/saml">SAML</a>
+    <ul class="usa-sidenav-sub_list">
+      <li>
+        <a class="{% if page.url == '/saml/' %}usa-current{% endif %}" href="{{ site.baseurl }}/saml">Endpoints</a>
```
Comment (jessieay) [inline _includes/sidenav.html]: I am surprised there isn't an easier way to do this in jekyll...

File: _pages/openid-connect.md [hunk 2]
```diff
@@ -283,12 +283,12 @@ GET https://idp.int.login.gov/openid_connect/userinfo
 Authorization: Bearer hhJES3wcgjI55jzjBvZpNQ
 ```
 
-### User Info Response
+### User info response
```
Comment (jessieay) [inline _pages/openid-connect.md]: sentence case ftw! (is that what this is called?)

File: _pages/register.md [hunk 3]
```diff
@@ -1,8 +1,11 @@
 ---
-title: Registering your Service Provider
+title: Register Your Service Provider
+permalink: /register/
+redirect_from:
+  - /registering-your-sp/
 ---
 
-# Registering your Service Provider
+# Register Your Service Provider
```
Comment (jessieay) [inline _pages/register.md]: sentence case-ify?

File: _pages/saml.md [hunk 4]
```diff
@@ -20,7 +22,7 @@ In this guide:
 
 ## Getting Started
```
Comment (jessieay) [inline _pages/saml.md]: `%s/Started/started`

File: _includes/sidenav.html (hunk 1 above)
Comment (zachmargolis) [inline _includes/sidenav.html]: by "this" you mean highlighting the current page? Yeah :[

File: _pages/openid-connect.md (hunk 2 above)
Comment (zachmargolis) [inline _pages/openid-connect.md]: Yeah, @blacktm pointed me at the 18f style guide for this: https://pages.18f.gov/content-guide/capitalization/

File: _pages/register.md (hunk 3 above)
Comment (zachmargolis) [inline _pages/register.md]: Eek, yes, will fix

File: _pages/saml.md (hunk 4 above)
Comment (zachmargolis) [inline _pages/saml.md]: Eek thanks!

File: _pages/register.md (hunk 3 above)
Comment (zachmargolis) [inline _pages/register.md]: Fixed: 5d372544c708b9446548a06b43fb5a7d3e416582

File: _pages/saml.md (hunk 4 above)
Comment (zachmargolis) [inline _pages/saml.md]: Fixed: 5d372544c708b9446548a06b43fb5a7d3e416582

**Comment by zachmargolis** [comment]
Does anybody have any additional feedback? Would love to get this merged soon.

**Pull request merged**